pypose/lietensor/lietensor.py

```diff
@@ -27,6 +32,17 @@
                      'index_select', 'masked_select', 'index_copy', 'index_copy_',
                      'select', 'select_scatter', 'index_put','index_put_', 'copy_']
 
+_size_any_t = Union[float, Tuple[float, ...], torch.Tensor]
+_penta = _ntuple(5, "_penta")
+
+def to_tuple(x):
+    out = tuple()
+    for i in x:
+        if not isinstance(i, collections.abc.Iterable):
+            out += (i,)
+        else:
+            out += tuple(i)
+    return out
```

Comment:
this function is unnecessary, remove it.
`size` is always a tuple in your codes.

Reply:
no, for `x = pp.SO3([0, 0, 0, 1])` , x.lshape is (torch.size([]), ).
How can this be processed the same as (int, ) ?

Reply:
it is taken as empty tuple, should be no problem, did you try?